youtube_scraper/scraper.py

```diff
@@ -7,29 +7,32 @@ class YoutubeScrape(object):
     def __init__(self, soup):
         """ Initialize and scrape """
         self.soup = soup
-        self.title = self.parse_string('.watch-title')
-        self.poster = self.parse_string('.yt-user-info')
-        self.views = self.parse_int('.watch-view-count')
-        self.published = self.parse_string('.watch-time-text')
+        self.title = self.parse_string('.watch-title', 0)
+        self.poster = self.parse_string('.yt-user-info', 0)
+        self.views = self.parse_int('.watch-view-count', 0)
+        self.published = self.parse_string('.watch-time-text', 0)
         self.published = re.sub(r'(Published|Uploaded) on', '', self.published).strip()
-        self.like = self.parse_int('#watch-like')
-        self.dislike = self.parse_int('#watch-dislike')
+        self.like = self.parse_int('.yt-uix-clickcard', 4)
+        self.dislike = self.parse_int('.yt-uix-clickcard', 5)
 
-    def parse_int(self, selector):
+    def parse_int(self, selector, pos):
```

Comment:
Great thanks! Can you make these two `pos` arguments be kwargs and default them to 0?
ie `def parse_int(self, selector, pos=0):`

Reply:
Hi! I made also this change. 
Thanks,

Mattia